src/sounds.py

```diff
@@ -0,0 +1,80 @@
+import parselmouth
+
+import argparse
+import pathlib
+import math
+import os
+
+class Sound:
+    def __init__(self, note, timestamp, duration):
+        self.note = note
+        self.timestamp = timestamp
+        self.duration = duration
+
+    @property
+    def symbol(self):
+        if self.note == None:
+            return 'None'
+        symbols = ['C', 'C#', 'D', 'D#', 'E', 'F', 'F#', 'G', 'G#', 'A', 'A#', 'H']
+        return symbols[self.note]  
+
+    def __str__(self):
+        return f"{round(self.timestamp, 3)}: {self.symbol}\n"
+
+    def __ne__(self, other):
```

Comment:
I made a mistake in last review. You should always override `__eq__(self, other)` method.

Reply:
Done